Modules/_testclinic.c

```diff
@@ -1193,6 +1193,190 @@ clone_with_conv_f2_impl(PyObject *module, custom_t path)
 }
 
 
+/*[clinic input]
+output push
+destination deprstar new file '{dirname}/clinic/_testclinic_depr_star.c.h'
+output everything deprstar
+#output methoddef_ifndef buffer 1
```

Comment:
The output directive has a bug that prevents us from specifying the stack index. We should fix that in a separate PR.